proposals/2022-04-upgrade-v2-0-3-min-commission-rate/submit.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+panacead tx gov submit-proposal software-upgrade v2.0.3 \
+	--upgrade-time "2022-05-11T07:00:00Z" \
+	--node https://rpc.gopanacea.org:443 \
+	--chain-id panacea-3 \
+	--from panacea1ewugvs354xput6xydl5cd5tvkzcuymkejekwk3 \
+	--fees 1000000umed \
+	--gas auto \
+	-b block \
+	--deposit "10000000000umed" \
+	--title "Upgrade the chain to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%" \
+	--description "### Proposal
+	
+This proposal is for upgrading the Panacea mainnet 'panacea-3' from v2.0.2 to v2.0.3 for introducing a min-commission-rate parameter and setting it to 3%.
+
+A signaling proposal for this had been already approved: https://www.mintscan.io/medibloc/proposals/2.
+For more details about the motivation of introducing the min-commission-rate parameter, please see the signaling proposal.
+
+
+### What the chain upgrade contains
+
+Please see a release note: https://github.com/medibloc/panacea-core/releases/tag/v2.0.3
+
+
+### Required actions by node operators
+
+If this SoftwareUpgrade proposal is approved, the state machine of the 'panacead' will be stopped as soon as a proposed block time '2022-05-11 07:00:00 UTC' is reached.
+It means that new blocks will not be produced until the 'panacead' daemon is restarted with the new version.
+
+Then, please replace the old 'panacead' binary with the new one and restart the daemon by the following guide.
```

Comment:
```suggestion
Hence, please replace the old `panacead` binary with the new one and restart the daemon by following the guide below.
```

Reply:
71af0d9d497b0a5c5971b48ca93fb0b42c4edeaa Thank you. I applied your suggestion except the backtick. In shell script (`*.sh`), the backtick shouldn't be used in string literals.